### PR TITLE
Session chart

### PR DESCRIPTION
**Pull request by katemacleod**
Requires the functionality included in #971

### CODE REVIEW

File: timesketch/frontend/src/components/SketchExploreSessionChart.vue [hunk 1]
```diff
@@ -0,0 +1,308 @@
...
+            }
+        }
+        else {
+            this.message = 'Show Session Chart'
+            this.selectedSessions = []
+            this.spec = '{}'
+            this.selectedType = ''
+            this.selectedID = ''
+            this.showTimeRange = false
+            this.startTimeRange  = Date.now() - 31556952000,
```
Comment (berggren) [inline timesketch/frontend/src/components/SketchExploreSessionChart.vue]: This is a year, right? Can we make this a constant to make it more descriptive?

File: timesketch/frontend/src/components/SketchExploreSessionChart.vue [hunk 2]
```diff
@@ -0,0 +1,308 @@
...
+            this.timeRangeMessage = ''
+        }
+    },
+
+    toggleTimeRange: function () {
+        this.showTimeRange = !this.showTimeRange
+    },
+    
+    updateTimeRange: function (event) {
+        if(/\d{1,2}\/\d{1,2}\/\d{4}/.test(event.target.value)) {
```
Comment (berggren) [inline timesketch/frontend/src/components/SketchExploreSessionChart.vue]: Same here, can you put the RE in a constant with a descriptive name to make this a bit more readable?

File: timesketch/frontend/src/components/SketchExploreSessionChart.vue [hunk 3]
```diff
@@ -0,0 +1,308 @@
...
+                if (session.end_timestamp < extended_end) {
+                    session.end_timestamp = extended_end
+                }
+            }
+        }
+        this.processedSessions = processedSessions
+    },
+
+    getVegaSpec: function () {
+        const WIDTH = 900
```
Comment (berggren) [inline timesketch/frontend/src/components/SketchExploreSessionChart.vue]: Can we make this responsive? Something like:
```
        spec.config.view.width = this.$refs.vegaChart.$el.offsetWidth
        spec.config.autosize = { type: 'fit', contains: 'padding' }
```

Comment (berggren) [reply timesketch/frontend/src/components/SketchExploreSessionChart.vue]: Totally fine to just add a #TODO here and we can do that at a later stage.

Comment (katemacleod) [reply timesketch/frontend/src/components/SketchExploreSessionChart.vue]: Added a TODO

**Pull request merged**